x-pack/test/observability_functional/apps/observability/alerts/workflow_status.ts

```diff
@@ -28,61 +30,61 @@ export default ({ getService }: FtrProviderContext) => {
 
     it('is filtered to only show "open" alerts by default', async () => {
       await retry.try(async () => {
-        const tableRows = await observability.alerts.getTableCellsInRows();
-        expect(tableRows.length).to.be(12);
+        const tableRows = await observability.alerts.common.getTableCellsInRows();
+        expect(tableRows.length).to.be(OPEN_ALERTS_ROWS_COUNT);
```

Comment:
I think Felix left this as a comment in another PR that Kerry made, that perhaps it would be best if we packed this constant into the same variable that holds the data range for our archived data? Since this number is tightly dependent on both the date range and the actual archive

Reply:
Ok I see what you mean. This makes total sense. I could add a new key on the existing object and export it:

```
export const DATE_WITH_DATA = {
  rangeFrom: '2021-09-01T13:36:22.109Z',
  rangeTo: '2021-09-03T13:36:22.109Z',
  numRows: 12,
};
```

If I reuse the existing constant, the extra numRows key will be appended in the urlbar, but this doesn't cause any problem. The tests still keep running without any issues. So is it fine to add the key as I shown above?

Reply:
As discussed I will create a new constant:

```
const ARCHIVE_DATA = {
  dateRange: {
    rangeFrom: '2021-09-01T13:36:22.109Z',
    rangeTo: '2021-09-03T13:36:22.109Z',
  },
  numOfAlerts: 12,
};
```

Reply:
Do you mind if I leave this part out of this PR for now? Above constant lies at the moment in a service folder. When I exported this constant and imported in the corresponding test file, the test runner completely crashed out.